TypeCobol/Compiler/Parser/CodeElementBuilder/CobolExpressionsBuilder.cs

```diff
@@ -1536,15 +1536,20 @@ internal ReceivingStorageArea CreateNumericStorageArea(CodeElementsParser.Numeri
 		}
 
 		internal ReceivingStorageArea CreateAlphanumericStorageArea(CodeElementsParser.AlphanumericStorageAreaContext context)
-		{
-			var storageArea = new ReceivingStorageArea(StorageDataType.Alphanumeric,
-				CreateIdentifier(context?.identifier()));
+        {
+            var identifier = CreateIdentifier(context?.identifier());
+            if (identifier == null)
+            {
+                return null;
+            }
+            var storageArea = new ReceivingStorageArea(StorageDataType.Alphanumeric, identifier);
```

Comment:
Ok this prevent the null identifier to be passed to the `ReceivingStorageArea` constructor but there are other usages where null could be passed. Should we change all of them in this PR ?

Reply:
I suggest to only fix this NRE in this PR, so we can advance on the related issue #1835.
We can use another issue to fix all potentiel NRE.

Reply:
@delevoye Could you please check usages of `CreateAlphanumericStorageArea` ? Now that it may return null, we must ensure that we have not created other potential null ref exceptions...

Reply:
The other case of potential NRE has been deleted as it was a wrong information in most cases.